CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unversioned
 
 - Minor: Delete all but the last 5 crashdumps on application start. (#4392)
+- Minor: Added `/banid` command that allows banning by userID. (#4411)
```

Comment:
```suggestion
- Minor: Added `/banid` command that allows banning by user ID. (#4411)
```

Reply:
none of the antonyms for `concatenate` are very exciting 😢